Formula/v/vcctl.rb

```diff
@@ -0,0 +1,19 @@
+class Vcctl < Formula
+  desc "Cloud Native Batch System (Project under CNCF)"
+  homepage "https://volcano.sh"
+  url "https://github.com/volcano-sh/volcano/archive/refs/tags/v1.12.2.tar.gz"
+  sha256 "07f89c901336bf9a85c0984c255d4600852f7465de4878bb0d530543bb7dfe97"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    system "make", "vcctl"
+    bin.install "_output/bin/vcctl"
+  end
+
+  test do
+    system bin/"vcctl", "--help"
```

Comment:
need substantial test

Reply:
added a few tests [similar](https://github.com/Homebrew/homebrew-core/blob/6f7da2a2e393227cdae167140ef26bb1e3834f44/Formula/k/kubernetes-cli.rb#L49-L54) to `kubernetes-cli`